### PR TITLE
replace path with filepath so sync works properly on windows machines

### PR DESCRIPTION
**Pull request by spf13**

### CODE REVIEW

File: fsync.go [hunk 1]
```diff
@@ -85,7 +86,7 @@ func (s *Syncer) Sync(dst, src string) error {
 // SyncTo syncs srcs files or directories into to directory.
 func (s *Syncer) SyncTo(to string, srcs ...string) error {
 	for _, src := range srcs {
-		dst := path.Join(to, path.Base(src))
```
Comment (mostafah) [inline fsync.go]: Shouldn’t `path.Base` be changed to `filepath.Base` too?

Comment (spf13) [inline fsync.go]: Oh.. good call. It probably should, but it worked fine for me without doing that.

Comment (mostafah) [inline fsync.go]: Alright then. I change that in another commit myself.

**Comment by mostafah** [comment]
Thank you for the fix.

**Pull request merged**